go.sum

```diff
@@ -1,13 +1,17 @@
 github.com/Kong/go-pdk v0.6.0 h1:KZZKGYK5NbTIMQa5P1IRldMvA5/LJEoHhrsOQlVCPHo=
 github.com/Kong/go-pdk v0.6.0/go.mod h1:SrAne3YN4a7pNrlJ6Fb6xwa6kOhV2vDY/mqpb3X0fs4=
```

Comment:
let's do a `go mod tidy` to get rid of old dependencies

Reply:
AFAICT, `go mod tidy` removes dependencies from `go.mod`, but not from `go.sum`.  The latter isn't a "lock file", and seems the Go developers are ok with letting it grow indefinitely.